README.md

```diff
@@ -8,4 +8,13 @@ This is an example app developed for a series of articles about Rails API apps a
 
 It is recommended you start with part 0, located at https://paweljw.github.io/2017/07/rails-5.1-api-with-vue.js-frontend-part-0-stack-choices/.
 
+## Getting Started
+
+Run the setup script and start the server
+
+```
+./bin/setup
```

Comment:
Do you think it might be reasonable to tell people who don't have Postgres running locally and are instead following the articles that they need to uncomment the lines in the copied over database.yml and also run `docker-compose up`?

Reply:
Yeah, I think that makes sense...just remind them to make any changes that relate to their personal database setup.